Front/src/Components/Sections/Infraestructure/item.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box } from "@mui/material"
+import colors from "../../../Styles/colors"
+
+type ItemData = {
```

Comment:
TItemData

para types sempre iniciar com o "T" no inicio, quando for criar interfaces inicia o nome com "I"